src/main/java/appeng/container/slot/SlotCraftingTerm.java

```diff
@@ -232,17 +232,17 @@ public ItemStack craftItem( final EntityPlayer p, final ItemStack request, final
 		return null;
 	}
 
-	public boolean preCraft( final EntityPlayer p, final IMEMonitor<IAEItemStack> inv, final ItemStack[] set, final ItemStack result )
+	private boolean preCraft( final EntityPlayer p, final IMEMonitor<IAEItemStack> inv, final ItemStack[] set, final ItemStack result )
 	{
 		return true;
 	}
 
-	public void makeItem( final EntityPlayer p, final ItemStack is )
+	private void makeItem( final EntityPlayer p, final ItemStack is )
 	{
 		super.onPickupFromSlot( p, is );
 	}
 
-	public void postCraft( final EntityPlayer p, final IMEMonitor<IAEItemStack> inv, final ItemStack[] set, final ItemStack result )
+	private void postCraft( final EntityPlayer p, final IMEMonitor<IAEItemStack> inv, final ItemStack[] set, final ItemStack result )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)